Jenkinsfile

```diff
@@ -0,0 +1,20 @@
+#!groovy
+try {
+    node("ecs") {
+        stage("Checkout") {
+            checkout scm
+        }
+        stage("Install dependencies") {
+            sh("yarn install")
+        }
+        stage("Tests") {
+            sh("yarn test")
+        }
+        stage('Finish') {
+            echo("Done. KTKSBAI!")
```

Comment:
hahahaha I forgot about that 😄 

Now I am not sure which one is the correct one!!! Is this real life?